src/util/file-filter.js

```diff
@@ -45,9 +43,10 @@ export class FileFilter {
       '**/node_modules/**/*',
     ],
     ignoreFiles = [],
-    sourceDir,
+    sourceDir = '.',
```

Comment:
There are some tests using `FileFilter` without `sourceDir`. This would force them to explicitly use `sourceDir: '.'`. Maybe it is a good thing?